engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/AbstractConfluentKafkaAvroDeserializer.scala

```diff
@@ -19,31 +19,31 @@ abstract class AbstractConfluentKafkaAvroDeserializer extends AbstractKafkaAvroD
 
   protected lazy val decoderFactory: DecoderFactory = DecoderFactory.get()
 
-  override protected def deserialize(topic: String, isKey: java.lang.Boolean, payload: Array[Byte], readerSchema: Schema): AnyRef = {
+  protected def deserialize(topic: String, isKey: java.lang.Boolean, payload: Array[Byte], readerSchema: RuntimeSchemaData): AnyRef = {
     val buffer = ConfluentUtils.parsePayloadToByteBuffer(payload).valueOr(ex => throw ex)
     read(buffer, readerSchema)
   }
 
-  protected def read(buffer: ByteBuffer, expectedSchema: Schema): AnyRef = {
+  protected def read(buffer: ByteBuffer, expectedSchema: RuntimeSchemaData): AnyRef = {
```

Comment:
readerSchema -> readerSchemaData

Reply:
Fixed in all places. You don't need to add comments in all places ;)